### PR TITLE
Generates payment QR codes internally instead of using an external service

### PR DESCRIPTION
**Pull request by jenrik**

### CODE REVIEW

**Review by cogitantium — APPROVED** [review]
Nice :ok_hand: Made some minor changes;
- Restructured into util.py and forms.py
- Removed printf debug statement
- Enlargened QR codes to 300x300px

File: stregsystem/views.py [hunk 1]
```diff
@@ -288,3 +290,21 @@ def batch_payment(request):
         "select2_js": settings.SELECT2_JS,
         "select2_css": settings.SELECT2_CSS,
     })
+
+
+def qr_payment(request):
+    form = QRPaymentForm(request.GET)
+    if not form.is_valid():
+        return HttpResponseBadRequest("Invalid input for MobilePay QR code generation")
```
Comment (jenrik) [inline stregsystem/views.py:298]: We should probably tell people which field they messed up or forgot

File: stregsystem/views.py [hunk 2]
```diff
@@ -288,3 +290,21 @@ def batch_payment(request):
...
+        'phone': '90601',
+        'comment': form.cleaned_data.get('member')
+    }
+
+    if form.cleaned_data.get("amount") is not None:
+        query['amount'] = form.cleaned_data.get("amount")
+
+    data = 'mobilepay://send?{}'.format(urllib.parse.urlencode(query))
+
+    return qr_code(request, data)
```
Comment (cogitantium) [inline stregsystem/views.py:310]:
```suggestion
    return qr_code(data)
```

File: stregsystem/views.py (hunk 1 above)
Comment (cogitantium) [reply stregsystem/views.py:298]: The users do not enter this information themselves. This is given by the Django template tags, e.g. `{{ data.value }}`.

**Review by cogitantium — COMMENTED** [review]
Fixing qr_code call in views

File: stregsystem/views.py (hunk 1 above)
Comment (jenrik) [reply stregsystem/views.py:298]: Users include current and future developers

**Pull request merged**